editor/editor.tsx

```diff
@@ -866,26 +851,34 @@ function Editor(props: RouteComponentProps) {
                         </span>
                         <span
                             title={
-                                code.length <= LIMIT_CLIPBOARD_LEN
+                                stringifySpec(goslingSpec || '').length <= LIMIT_CLIPBOARD_LEN
```

Comment:
I see `stringifySpec(goslingSpec || '')` is used multiple times. Would it make sense to let `stringifySpec` deal with `undefined`, something like?:

```js
const stringifySpec = (spec: string | gosling.GoslingSpec | undefined): string => {
    if (!spec) return '';
    else if (typeof spec === 'string') return spec;
    else return stringify(spec);
};
```

Reply:
Good point!